pysmt/smtlib/parser/parser.py

```diff
@@ -1304,6 +1305,8 @@ def __init__(self, environment=None, interactive=False):
                                 self._operator_adapter(self._ext_rotate_left)
         self.interpreted["ext_rotate_right"] =\
                                 self._operator_adapter(self._ext_rotate_right)
+        mgr = self.env.formula_manager
+        self.interpreted['bv2int'] = self._operator_adapter(mgr.BVToNat)
```

Comment:
I think this should be
```python
self.interpreted['bv2nat`] = self._operator_adapter(mgr.BVToNat)
```
AFAIK `bv2int` is z3-specific, but we might support it as an alias of `bv2nat` (see https://github.com/Z3Prover/z3/issues/1252)

Reply:
This is an extension of the main parser to deal with z3. z3 outputs bv2int when dumping SMT-LIB.

Reply:
This is an extension of the main parser to deal with z3. z3 outputs bv2int when dumping SMT-LIB.